acme.go

```diff
@@ -330,7 +330,7 @@ func (c *Client) Authorize(ctx context.Context, domain string) (*Authorization,
 	if err := json.NewDecoder(res.Body).Decode(&v); err != nil {
 		return nil, fmt.Errorf("acme: invalid response: %v", err)
 	}
-	if v.Status != StatusPending {
+	if v.Status != StatusPending && v.Status != StatusValid {
```

Comment:
Sorry, I feel dumb these days. Should've got it sooner when you said "upstream".